documentation/src/docs/streamcuts.md

```diff
@@ -65,30 +69,45 @@ ReaderGroupConfig.builder()
                 .build();
 ```
 
-The below API can be used to reset an existing ReaderGroup with a new ReaderGroup configuration instead creating a
-ReaderGroup.
+The below API can be used to reset an existing Reader Group with a new Reader Group configuration instead creating a Reader Group.
 ```
 /*
  * ReaderGroup API used to reset a ReaderGroup to a newer ReaderGroup configuration.
  */
 io.pravega.client.stream.ReaderGroup.resetReaderGroup(ReaderGroupConfig config)
+```
+
+## StreamCut with Stream Manager
+
+`StreamCut` representing the current head and current tail of a stream can be obtained using the `StreamManager` API. The `getStreamInfo(String scopeName, String streamName)` API on `StreamManager` ensures that the `getStreamInfo()` API can be used outside the `BatchClient`.
```

Comment:
Both the sentences could be combined . No need to mention the BatchClient.
something like the following...
```
`StreamCut`s representing the current head and current tail of the stream can be obtained by the `getStreamInfo(String scope, String streamName ) API on the the `StreamManager`
```